main.go

```diff
@@ -34,3 +36,16 @@ func main() {
 		fmt.Println("http.http.ListenAndServe Error", err)
 	}
 }
+
+func getHTTPDownloadURL(p github.PushPayload) string {
+	return "git::" + p.Repository.URL
+}
+
+func downloadRepo(downloadURL string) {
+	err := getter.Get("./repo", downloadURL)
+	if err != nil {
+		fmt.Println("Error downloading repo:", err)
+		return
+	}
+	fmt.Println("Repo downloaded from:", downloadURL)
```

Comment:
Should we use a better logging library?

Reply:
switching to the standard go log package